src/auth/feedback-secret.guard.ts

```diff
@@ -0,0 +1,24 @@
+import {
+  Injectable,
+  CanActivate,
+  ExecutionContext,
+  UnauthorizedException,
+} from '@nestjs/common';
+import { Env } from 'src/env';
+
+@Injectable()
+export class FeedbackSecretGuard implements CanActivate {
+  constructor() {}
```

Comment:
PullProofAI

The constructor in the `FeedbackSecretGuard` class is empty, which is unnecessary.
```diff
-   constructor() {}
```

#### Suggested Fix

Remove the empty constructor as it serves no purpose in this context.

This was the only issue found in the provided code snippet.


<!-- This is an auto-generated comment by PullProofAI -->